### ISSUE TITLE
database-properties-generate() is not generating all path-namespaces from model in e2e

### ISSUE THREAD

**Issue by bsrikan** [marklogic/entity-services#342]
Model valid-ref.json in e2e has 2 namespaces with prefix 'cust' and 'ord' but only one is generated in database-properties-generate()

```
{
"database-name": "%%DATABASE%%", 
"schema-database": "%%SCHEMAS_DATABASE%%", 
"path-namespace": [
{
"prefix": "cust", 
"namespace-uri": "http://marklogic.com/customer"
}, 
{
"prefix": "es", 
"namespace-uri": "http://marklogic.com/entity-services"
}
```

**Comment by grechaw** [comment]
I think your working copy of valid-ref-2 is not on github yet?  I'd like to examine it, as it seemed like other similar model descriptors were working OK.

**Comment by bsrikan** [comment]
Attached those models herewith.
[e2e_ns_Models.zip](https://github.com/marklogic/entity-services/files/1265026/e2e_ns_Models.zip)

**Comment by grechaw** [comment]
thanks ill check them out.

**Comment by grechaw** [comment]
OK -- not a bug.

I found where in the code I am neglecting to include path range indexes for namespaces.   Turns out behavior is correct.

If there are element range indexes, then the namespace information is included right in the element-range-index block.  It doesn't need an entry in path-namespaces to resolve it.

So for those types with element-range-index, but not a path-index, there will be no path-namespace definition.

Back to you for verification
